change/@fluentui-react-provider-74fe7dfa-c649-49cb-b118-0db8cc535e57.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29894&commitId=2a68ec9be78a9d0048acad49e2cb076edb9a7666&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|200| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>